modules/library/src/main/scala/zio/elasticsearch/HttpElasticExecutor.scala

```diff
@@ -305,33 +415,27 @@ private[elasticsearch] final class HttpElasticExecutor private (config: ElasticC
       _    <- logDebug(s"[es-res]: ${resp.show(includeBody = true, includeHeaders = true, sensitiveHeaders = Set.empty)}")
     } yield resp
 
-  private def createElasticException(response: Response[Either[String, String]]): ElasticException =
-    new ElasticException(
-      s"Unexpected response from Elasticsearch. Response body: ${response.body.fold(body => body, _ => "")}"
-    )
-
-  private def createElasticExceptionFromCustomResponse[A](
-    response: Response[Either[ResponseException[String, String], A]]
-  ): ElasticException =
-    new ElasticException(
-      s"Unexpected response from Elasticsearch. Response body: ${response.body.fold(body => body, _ => "")}"
-    )
-
-  private def getQueryParams(parameters: List[(String, Any)]): ScalaMap[String, String] =
-    parameters.collect { case (name, Some(value)) => (name, value.toString) }.toMap
-
 }
 
 private[elasticsearch] object HttpElasticExecutor {
 
-  private final val Bulk                  = "_bulk"
-  private final val Create                = "_create"
-  private final val DeleteByQuery         = "_delete_by_query"
-  private final val Doc                   = "_doc"
-  private final val Search                = "_search"
-  private final val Scroll                = "scroll"
-  private final val ScrollDefaultDuration = "1m"
-  private final val ScrollId              = "scroll_id"
+  private final val Bulk                     = "_bulk"
+  private final val Create                   = "_create"
+  private final val DeleteByQuery            = "_delete_by_query"
+  private final val Doc                      = "_doc"
+  private final val KeepAlive                = "keep_alive"
+  private final val KeepAliveDefaultDuration = "1m"
+  private final val PointInTime              = "_pit"
+  private final val Search                   = "_search"
+  private final val Scroll                   = "scroll"
+  private final val ScrollDefaultDuration    = "1m"
+  private final val ScrollId                 = "scroll_id"
+
+  private[elasticsearch] final case class PointInTimeResponse(id: String)
+  object PointInTimeResponse {
```

Comment:
Add new line here.

Reply:
Should we move `Response` somewhere else?

Reply:
Maybe, but it is only used here and is very small class, i think it might be ok to keep it here.